hack/component-versions.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Maybe re-generate this file using the Go code, that way it has the "generated file" warning here.